Automattic/wp-calypso
# Migrate remove button styles

#### Changes proposed in this Pull Request

Migrate the remove button styles used on featured images.

#### Testing instructions

1. Edit a post
2. Set a featured image for the post
3. Verify that the remove button is properly styled and overlaying the featured image

##### Unstyled
<img width="252" alt="screen shot 2018-10-03 at 12 16 05 am" src="https://user-images.githubusercontent.com/10561050/46371440-f3205800-c6ba-11e8-8525-9bdca15d93d7.png">

##### Styled
<img width="251" alt="screen shot 2018-10-03 at 12 17 59 am" src="https://user-images.githubusercontent.com/10561050/46371442-f4518500-c6ba-11e8-8bf5-d34f3944447b.png">


#27515
